lib/glific/reports.ex

```diff
@@ -139,6 +139,52 @@ defmodule Glific.Reports do
     """
   end
 
+  defp get_flow_name_and_group_name_sql(flow_id, group_id) do
+    """
+    SELECT name AS result FROM flows WHERE id = #{flow_id}
+    UNION
+    SELECT label AS result FROM groups WHERE id = #{group_id};
+    """
+  end
+
+  defp get_flow_name_and_group_name(flow_id, group_id) do
+    get_flow_name_and_group_name_sql(flow_id, group_id)
+    |> Repo.query!([])
+    |> then(& &1.rows)
+  end
+
+  @doc """
+    gets data for the broadcast table
+  """
+  @spec get_broadcast_data(non_neg_integer()) :: list()
+  def get_broadcast_data(org_id) do
+    query_data =
+      get_broadcast_query(org_id)
+      |> Repo.query!([])
+
+    Enum.map(query_data.rows, fn
+      [flow_id, group_id, started, nil] ->
+        [[name], [label]] = get_flow_name_and_group_name(flow_id, group_id)
+        [name, label, Timex.format!(started, "%H:%M, %d-%m-%Y", :strftime), "Not Completed Yet"]
+
```

Comment:
<img width="503" alt="image" src="https://github.com/glific/glific/assets/40158831/7f585a1f-3ed8-4749-b32e-16569768a6f3">

Right now time format is `07:58, 20-07-2023` can u change this to `20-07-2023 07:58 AM`. This way it will look a bit better